proposals/incubation/cert-manager.md

```diff
@@ -0,0 +1,91 @@
+
+### Production Use
+
+> Document that it is being used successfully in production by at least three independent end users
+
+cert-manager is used in production by _many_ more than three independent end users. Several major cloud native projects including [Istio](https://istio.io/latest/docs/ops/integrations/certmanager/), the [Kubernetes Cluster API](https://cluster-api.sigs.k8s.io/developer/guide.html#cert-manager) and [Knative](https://knative.dev/development/install/serving/installing-cert-manager/) either suggest or require that cert-manager be installed.
+
+According to the [companies-table DevStats dashboard](https://certmanager.devstats.cncf.io/d/5/companies-table?orgId=1) over a thousand different companies (plus ~1330 "Independents") have made some kind of contribution (defined as somebody who made a review, comment, commit, created PR or issue) to cert-manager. That implies that the floor of cert-manager's reach can be measured in the thousands, but there could be many more users who happily use cert-manager without contributing back or raising GitHub issues.
+
+One prominent example of a cert-manager user might be the CNCF themselves, using cert-manager [in the DevStats project](https://github.com/cncf/devstats#architecture)!
+
+In addition we might measure some production use through the many large companies who have seen fit to create addons for cert-manager to help it better integrate into their environments. That includes (at the time of writing):
+
+- at least [49 public repositories](https://github.com/topics/cert-manager-webhook) implementing an external ACME DNS-01 solver
+- at least [8 public repositories](https://cert-manager.io/docs/configuration/external/) implementing complete external issuers for certificates
+
+### Committer Numbers
+
+> Have a healthy number of committers
+
+cert-manager currently has 9 maintainers across 3 organisations. Community contributions from many different organizations are very common and gladly received.
+
+The [companies-table DevStats dashboard](https://certmanager.devstats.cncf.io/d/5/companies-table?orgId=1) is illustrative. We've seen:
+
+- 4 different companies which have made over a thousand contributions each,
+- 8 different companies which have made over a hundred contributions each,
+- and over a thousand different companies which have contributed something to the project.
+
+In 2021, cert-manager saw roughly 45 new first-time-contributors to the codebase. cert-manager v1.7 saw at least 7 different individual community contributors, and one of the major features in that release was [contributed by a community member](https://cert-manager.io/docs/release-notes/release-notes-1.7/#additional-certificate-output-formats).
+
+In the last 30 days at the time of writing, there were [15 different first-time contributors to the project](https://certmanager.devstats.cncf.io/d/52/new-contributors-table?orgId=1&from=now-30d&to=now).
```

Comment:
Can we say something more about the number of committers as well? The criteria is to have a healthy number of committers, rather than the number of companies.

Reply:
Works out at an average of 10 different committers per month over the last 6 months (and across most timescales, really); I've added some details :+1: